man/journalctl.xml

```diff
@@ -408,6 +408,14 @@
         printed.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-s</option></term>
+        <term><option>--strip</option></term>
+
+        <listitem><para>Strip each message so that only the first line is
```

Comment:
hmm, "strip" sounds like a weird name for this. Maybe call thus "--truncate-newline"?